io.openems.edge.evcs.alpitronic.hypercharger/test/io/openems/edge/evcs/hypercharger/MyConfig.java

```diff
@@ -82,7 +82,7 @@ public String modbus_id() {
 
 	@Override
 	public String Modbus_target() {
-		return ConfigUtils.generateReferenceTargetFilter(this.modbus_id(), this.modbus_id());
+		return ConfigUtils.generateReferenceTargetFilter(this.id(), this.modbus_id());
```

Comment:
Nach unseren Coding-Guidelines gehört das nicht hier in diesen Pull-Request (https://github.com/OpenEMS/openems/blob/develop/doc/modules/ROOT/pages/contribute/coding-guidelines.adoc?plain=1#L21)

Reply:
Ich bin jetzt nicht ganz sicher was ich hier machen soll. @sebastianasen hat im letzten Commit bevor er approved hat diese Änderungen in der Alpitronic gemacht. Soll ich die jetzt wieder rausnehmen?

> 
> Formatting & organising imports
> Add unit test
> Change priority of some tasks
> Use different event topic to call read and write handlers
> Remove EnergySession channel that stores the same value as Evcs/SessionEnergy
> 
> Unrelated:
> Minor changes for testing in Alpitronic

^^^^^^^

Reply:
Nein, kein Problem. Das kann hier drin bleiben - der Fix ist ja offensichtlich richtig. 👍